lib/karaokium/performances/performance.ex

```diff
@@ -24,4 +26,70 @@ defmodule Karaokium.Performances.Performance do
     |> cast(attrs, [:karaoke_id, :team_id, :song_id, :voting?])
     |> validate_required([:karaoke_id, :team_id, :song_id, :voting?])
   end
+
+  @doc """
+  Calculates a performance's score.
+
+  Uses [Wikipedia's first method](https://en.wikipedia.org/wiki/Quartile#Method_1) to calculate quartile values.
+  From these, determines the outliers and excludes such values from the final average calculation.
+
+  Preloads votes if they are not already loaded.
+  Returns 0 if there are no votes.
+
+  ## Examples
+
+      iex> votes = [%Karaokium.Polling.Vote{pontuation: 1} | List.duplicate(%Karaokium.Polling.Vote{pontuation: 9}, 5)]
+      iex> performance = %Karaokium.Performances.Performance{votes: votes}
+      iex> Karaokium.Performances.Performance.score(performance)
+      #Decimal<9.00>
+
+      iex> performance = %Karaokium.Performances.Performance{votes: []}
+      iex> Karaokium.Performances.Performance.score(performance))
+      #Decimal<0>
+
+      iex> performance = %Karaokium.Performances.Performance{}
+      iex> Karaokium.Performances.Performance.score(performance))
+      #Decimal<0>
```

Comment:
@nelsonmestevao Not sure how to exemplify that a performance with not yet loaded votes doesn't necessarily return 0.

Reply:
I think you don't need this example since you have a note in the documentation saying it preloads votes if not preloaded.